django_event_sourcing/registers.py

```diff
@@ -67,8 +72,20 @@ def handle(self, event, skip_side_effects=False):
             if skip_side_effects or handler_log.failed:
                 continue
 
-            for side_effect in self.side_effects[handler]:
+            for registered_side_effect in self.side_effects[handler]:
+                # Check the condition here
+                condition = registered_side_effect.condition
```

Comment:
```suggestion
                condition_class = registered_side_effect.condition
```

I think this will make it clearer that it's not an instance of a condition at this point.